pkgdb2/ui/packages.py

```diff
@@ -349,9 +351,10 @@ def package_request_edit(package, action_id):
         )
 
     # Check user is the pkg/pkgdb admin
+    pkg_admin = pkgdblib.has_acls(
+        SESSION, flask.g.fas_user.username, package, 'approveacls')
```

Comment:
Local tests showed it behaves correctly but I've added a second check in has_acls itself to cover the situation (and avoid querying the DB for nothing)